dev/doc/debugging.md

```diff
@@ -22,65 +22,12 @@ Debugging from Coq toplevel using Caml trace mechanism
-   byte-code version of `coqtop` must have been generated with `make byte`.
-
-   1. M-x camldebug
-   2. give the binary name bin/coqtop.byte
-   3. give ../dev/ocamldebug-coq
-   4. source db  (to get pretty-printers)
-   5. add breakpoints with C-x C-a C-b from the buffer displaying the ocaml
-      source
-   6. get more help from ocamldebug manual
-         run
-	 step
-         back
-         start
-	 next
-	 last
-	 print x (abbreviated into p x)
-	 ...
-   7. some hints: 
-
-   - To debug a failure/error/anomaly, add a breakpoint in
-     `Vernac.interp_vernac` (in `toplevel/vernac.ml`) at the with clause of the "try ... interp com
-     with ..." block, then go "back" a few steps to find where the
-     failure/error/anomaly has been raised
-   - Alternatively, for an error or an anomaly, add breakpoints in the middle  
-     of each of error* functions or anomaly* functions in lib/util.ml
-   - If "source db" fails, do a "make printers" and try again (it should build
-     top_printers.cmo and the core cma files).
-   - If you build Coq with an OCaml version earlier than 4.06, and have the 
-     OCAMLRUNPARAM environment variable set, Coq may hang on startup when run 
-     from the debugger. If this happens, unset the variable, re-start Emacs, and 
-     run the debugger again.
-
-Debugging with ocamldebug from the command line
-===============================================
-
-In the `coq` directory:
-1. (on Cygwin/Windows) Pass the `-no-custom` option to the `configure` script before building Coq.
-2. Run `make` (to compile the .v files)
-3. Run `make byte`
-4. (on Cygwin/Windows) Add the full pathname of the directory `.../kernel/byterun` to your bash PATH.
-   Alternatively, copy the file `kernel/byterun/dllcoqrun.dll` to a directory that is in the PATH.  (The
-   CAML_LD_LIBRARY_PATH mechanism described at the end of INSTALL isn't working.)
-5. Run `dev/ocamldebug-coq bin/coqtop.byte`  (on Cygwin/Windows, use `... bin/coqtop.byte.exe`)
-6. Enter `source db` to load printers
-7. Enter `set arguments -coqlib .` so Coq can find plugins, theories, etc.
-8. See the ocamldebug manual for more information.  A few points:
-   - use `break @ Printer 501` to set a breakpoint on line 501 in the Printer module (printer.ml).
-     `break` can be abbreviated as `b`.
-   - `backtrace` or `bt` to see the call stack
-   - `step` or `s` goes into called functions; `next` or `n` skips over them
-   - `list` or `li` shows the code just before and after the current stack frame
-   - `print <var>` or `p <var>` to see the value of a variable
-Note that `make byte` doesn't recompile .v files.  `make` recompiles all of them if there
-are changes in any .ml file--safer but much slower.
+Debugging with ocamldebug from Emacs or command line
+====================================================
+
+See build-system.dune.md#ocamldebug
+
+Remember that you should not use configure unless calling dune through the legacy build system.
```

Comment:
This is too strong, running configure with dune is supported and required in some cases; IMHO the correct wording is "you don't need to run configure when ..."